google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/StreamWriterTest.java

```diff
@@ -703,6 +710,16 @@ public void testWriterId()
     Assert.assertNotEquals(writer1.getWriterId(), writer2.getWriterId());
   }
 
+  @Test
+  public void testInitialization_operationKind() throws Exception {
```

Comment:
Do we need to duplicate all the tests into these two dimensions?

Reply:
Let's do this in a separate CL, my plan is to duplicate some of them